taskcluster/ci/server-megazord-assemble/kind.yml

```diff
@@ -0,0 +1,47 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+# release-publish: publish artifacts for successful nightly/release builds
+#
+# This task publishes a JSON file for the nightly/release builds. The JSON file is used by consumer
+# apps for app-services version bumps.
+#
+# This task only runs if build-summary succeeds, which implies that all other
+# tasks suceeded.  This task should not fail, since that could leave the
+# build artifacts in an inconsistent state.
```

Comment:
Just confirming that this comment is accurate? I don't see `build-summary` in `kind-dependencies` and from the changes to `build-summary/kind.yaml` it seems like it's the opposite, this job is a dependency of the build summary task

Reply:
Good catch, yes this patch sets up `build-summary` to depend on this task, not the other way around. I'm not sure whether the comment or the implementation is incorrect, but one of them should be fixed :)